src/test/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketWebhookClientImplTest.java

```diff
@@ -94,6 +100,15 @@ public void testRegisterWebhook() {
         BitbucketWebhook result = client.registerWebhook(request);
 
         assertThat(result.getEvents(), hasItems(repoRefEvent, mirrorSyncEvent));
+
+        Request recordedRequest = fakeRemoteHttpServer.getRequest(registerUrl);
+        Buffer b = new Buffer();
+        try {
+            recordedRequest.body().writeTo(b);
+            assertEquals("Request body not same as expected.", deleteWhitespace(normalizeSpace(requestBody)), new String(b.readByteArray()));
+        } catch (IOException e) {
```

Comment:
just add a `throws IOException` to method signature. This just becomes clutter. Tests can throw exceptions if needed.